documentation/modules/upgrading/proc-downgrade-kafka-kraft.adoc

```diff
@@ -0,0 +1,80 @@
+// This module is included in the following assemblies:
+//
+// assembly-downgrade.adoc
+
+[id='proc-downgrade-kafka-kraft-{context}']
+= Downgrading KRaft-based Kafka clusters and client applications
+
+[role="_abstract"]
+Downgrade a KRaft-based Strimzi Kafka cluster to an earlier version.
+When downgrading a KRaft-based Strimzi Kafka cluster to a lower version, like moving from {KafkaVersionHigher} to {KafkaVersionLower}, ensure that the `spec.kafka.metadataVersion` property in the Kafka configuration aligns with a version supported by the specified Kafka version (`spec.kafka.version`). 
+The metadata version for the Kafka version you are downgrading from must not be higher than the version you are downgrading to.
+
+NOTE: Consult the Apache Kafka documentation for information regarding the support and limitations associated with KRaft-based downgrades.
+
+.Prerequisites
+
+* The Cluster Operator is up and running.
+* Before you downgrade the Strimzi Kafka cluster, check the following for the `Kafka` resource:
+
+** `Kafka.spec.kafka.config` does not contain options that are not supported by the Kafka version being downgraded to.
+** `Kafka.spec.kafka.config` has a `spec.kafka.metadataVersion` that is supported by the Kafka version being downgraded to.   
+
+.Procedure
+
+. Update the Kafka cluster configuration.
++
+[source,shell,subs=+quotes]
+kubectl edit kafka <kafka_configuration_file>
+
+. Change the `Kafka.spec.kafka.version` and the `metadataVersion` version to specify the previous versions.
++
+For example, if downgrading from Kafka {KafkaVersionHigher} to {KafkaVersionLower}:
++
+[source,yaml,subs=attributes+]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    replicas: 3
+    metadataVersion: {KafkaMetadataVersionLower} # <1>
+    version: {KafkaVersionLower} # <2>
+    # ...
+----
++
+<1> Metadata version is supported by the Kafka version.
+<2> Kafka version is changed to the new version.
++
+NOTE: The value of `metadataVersion` must be a string to prevent it from being interpreted as a floating point number.
+
+. If the image for the Kafka version is different from the image defined in `STRIMZI_KAFKA_IMAGES` for the Cluster Operator, update `Kafka.spec.kafka.image`.
++
+See xref:con-versions-and-images-str[]
+
+. Save and exit the editor, then wait for rolling updates to complete.
++
+Check the update in the logs or by watching the pod state transitions:
++
+[source,shell,subs=+quotes]
+----
+kubectl logs -f _CLUSTER-OPERATOR-POD-NAME_ | grep -E "Kafka version downgrade from [0-9.]+ to [0-9.]+, phase ([0-9]+) of \1 completed"
+----
++
+[source,shell,subs=+quotes]
+----
+kubectl get pod -w
+----
++
+Check the Cluster Operator logs for an `INFO` level message:
++
+[source,shell,subs=+quotes]
+----
+Reconciliation #_NUM_(watch) Kafka(_NAMESPACE_/_NAME_): Kafka version downgrade from _FROM-VERSION_ to _TO-VERSION_, phase 1 of 1 completed
+----
```

Comment:
Maybe we prefer to say check resource status here like upgrades

Reply:
I wonder if these messages are still logged. I cannot find in the code.

Reply:
I removed all this and pointed to the resource status section.